docs/en/observability/inspect-panel.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[inspect-panel]]
+= Inspect
+
+The *Inspect* view in {kib} allows you to view information about all requests that were made to collect the data displayed on the current page.
+
+// Many requests go into building visualizations in {kib}.
+// For example, showing monitors in the {uptime-app} requests invloves...
+// getting all monitors 
+// ... 
+// ...
```

Comment:
👋  @shahzad31 @sqren can you help me articulate the benefit of this feature? Feel free to link to me to an existing issue/PR/doc that touches on the user need for the feature. (I did a bit of searching to find more info, but didn't find many details!)

Reply:
i think mostly it's used for debugging purpose during development, to check which queries are being executed by the page and if the response is as expected. Within uptime we also use it as a troubleshooting mechanism to ask customer to check what queries and responses they are getting in case some problem happens and they don't see the data.